assets/js/admin/course-video-progression-sidebar.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { __ } from '@wordpress/i18n';
+import { SelectControl, ToggleControl } from '@wordpress/components';
```

Comment:
I don't think we're using the `SelectControl`. Can it be removed from the import?